### DISCUSSION TITLE
tags editing quirks

### DISCUSSION THREAD

**Discussion by DrBruhman** [FakeFishGames/Barotrauma#13861] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

method that removes same tags removes word in differend tag...
example:
in:
![изображение](https://github.com/FakeFishGames/Barotrauma/assets/95565216/dfa3d4bc-7763-48d8-9704-c22fc60f14e9)
out:
![изображение](https://github.com/FakeFishGames/Barotrauma/assets/95565216/103ac31d-aac7-49ed-b4af-a08386d42620)
example 2:
in:
![изображение](https://github.com/FakeFishGames/Barotrauma/assets/95565216/927be9ed-c703-473b-8720-6b0686aa7dd0)
out:
![изображение](https://github.com/FakeFishGames/Barotrauma/assets/95565216/38236570-aeaa-42fa-a3df-3db785be9b16)
(so items with 2 or more predifined tags are working fine)

### Reproduction steps

subeditor: type "sampletag_engine" in engine item Tags textbox and observe that word engine gets removed from your tag

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### -

e

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
also my game crashed once without any logs when i opened modlist menu
```

**Comment by Regalis11** [comment]
Thank you for the report! I created [a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13877), we'll look into it!

**Comment by Rokvach** [comment]
Issue has been fixed and tested. Working correctly, closing discussion.
